App.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import './App.css';
+import Calculator from './components/calculator';
+
+// eslint-disable-next-line react/prefer-stateless-function
+class App extends React.Component {
```

Comment:
Great job @amklass11 🤝

- [ ] [Optional] I would suggest you change this component into a functional component too.